is-local-authenticator-archetype/src/main/resources/archetype-resources/src/main/java/__connector_name__Authenticator.java

```diff
@@ -0,0 +1,78 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package};
+
+import org.wso2.carbon.identity.application.authentication.framework.AbstractApplicationAuthenticator;
+import org.wso2.carbon.identity.application.authentication.framework.LocalApplicationAuthenticator;
+import org.wso2.carbon.identity.application.authentication.framework.context.AuthenticationContext;
+import org.wso2.carbon.identity.application.authentication.framework.exception.AuthenticationFailedException;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Authenticator of ${connector_name}
+ */
+public class ${connector_name}Authenticator extends AbstractApplicationAuthenticator
+        implements LocalApplicationAuthenticator {
+
+    private static Log log = LogFactory.getLog(${connector_name}Authenticator.class);
+
+    @Override
+    protected void initiateAuthenticationRequest(HttpServletRequest request,
+                                                 HttpServletResponse response, AuthenticationContext context)
+        throws AuthenticationFailedException {
+
+        // Use this to do any initializations to the authentication. eg. this may be used to redirect user to a page
+        // where he can submit the credentials
```

Comment:
```suggestion
        // Use this method to initiate the authentication request. Eg. this may be used to redirect user to a page
        // where for submitting credentials.
```